r/R/dplyr.R

```diff
@@ -540,6 +542,31 @@ arrow_stringr_string_replace_function <- function(FUN, max_replacements) {
   }
 }
 
+arrow_r_string_split_function <- function(FUN, reverse = FALSE, max_splits = -1){
+  function(x, split, fixed = FALSE, perl = FALSE, useBytes = FALSE){
+    
+    assert_that(length(split) == 1)
+    
+    # if !fixed but no regex metachars in split pattern, allow to proceed as split isn't regex
+    if(!fixed && contains_regex(split)){
+      stop("regular expression matching not supported in strsplit for Arrow", call. = FALSE)
```

Comment:
I suppose it's best to capitalize here since the message is a full sentence
```suggestion
      stop("Regular expression matching not supported in strsplit for Arrow", call. = FALSE)
```

Reply:
Also good to add parens after the function name, for clarity. The tidyverse style guide is big into always using parens after function names.